test/models/addressinfo.js

```diff
@@ -12,10 +9,11 @@ describe('AddressInfo', function() {
 
   describe('instantiation', function() {
 
-    var data = require('./sampleAddressFromInsight.json');
+    var data = JSON.stringify(require('./sampleAddressFromInsight.json'));
```

Comment:
It should still work with the JS object as well. Not just with the string. Do you want to test both?